src/Datadog.Trace/Tracer.cs

```diff
@@ -361,6 +366,130 @@ private static IStatsd CreateDogStatsdClient(TracerSettings settings, string ser
             return new Statsd(statsdUdp, new RandomGenerator(), new StopWatchFactory(), prefix: string.Empty, constantTags);
         }
 
+        private static void StartStandaloneAgentProcessesWhenConfigured()
+        {
+            try
+            {
+                var datadogConfigPath = Environment.GetEnvironmentVariable(ConfigurationKeys.AgentConfigPath);
+                string traceProcessArgs = null;
+                string dogStatsDArgs = null;
+
+                if (!string.IsNullOrWhiteSpace(datadogConfigPath))
+                {
+                    traceProcessArgs = $"--config \"{datadogConfigPath}\"";
+                }
+
+                _traceAgentMonitor = StartProcessWithKeepAlive(ConfigurationKeys.AgentPath, traceProcessArgs);
```

Comment:
Consider getting the env var values here and
- skip all the logic if they are not defined (tasking a `Task`, etc)
- pass the actual path to `StartProcessWithKeepAlive()` instead of the env var name